extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -548,7 +548,8 @@ class AmpAccordion extends AMP.BaseElement {
       if (newFocusIndex < 0) {
         newFocusIndex = newFocusIndex + this.headers_.length;
       }
-      const newFocusHeader = this.headers_[newFocusIndex];
+      const newFocusHeader =
+      dev().assertElement(this.headers_[newFocusIndex]);
```

Comment:
These `dev().assertX` calls are blockers because of the code size increases. We need to find a way to fix `devAssert` to get these type assertions and allow it to be eliminated from the build.

Reply:
The problem is that `devAssert` merely verifies the truishness of the argument, and knows nothing about its type.

Without this change:
```js
extensions/amp-accordion/0.1/amp-accordion.js:552: ERROR - actual parameter 1 of tryFocus$$module$src$dom does not match formal parameter
found   : Node
required: Element
      tryFocus(newFocusHeader);
               ^^^^^^^^^^^^^^
```

With `devAssert` instead of `dev().assertElement` (same error):
```js
extensions/amp-accordion/0.1/amp-accordion.js:552: ERROR - actual parameter 1 of tryFocus$$module$src$dom does not match formal parameter
found   : Node
required: Element
      tryFocus(newFocusHeader);
               ^^^^^^^^^^^^^^
```

Reply:
`headers_` should be changed to point to `Element` then?